rfc/src/rfcs/0010-quantifiers.md

```diff
@@ -0,0 +1,207 @@
+- **Feature Name:** Quantifiers
+- **Feature Request Issue:** [#2546](https://github.com/model-checking/kani/issues/2546) and [#836](https://github.com/model-checking/kani/issues/836)
+- **RFC PR:** [#](https://github.com/model-checking/kani/pull/)
+- **Status:** Unstable
+- **Version:** 1
+
+-------------------
+
+## Summary
```

Comment:
I would move this definitions to the section below, and just write 1-2 sentences on what this is proposing.

Add support to quantifiers in Kani to allow users to express properties about objects over an entire domain that satisfy a given condition.

Reply:
I re-wrote the summary and moved the text for the second section.